notifications_api_common/task.py

```diff
@@ -0,0 +1,57 @@
+from celery.exceptions import Ignore, Retry
+from celery.utils.time import get_exponential_backoff_interval
+from vine.utils import wraps
+
+from .models import NotificationsConfig
+
+
+def add_autoretry_behaviour(task, **options):
+    """
+    Adapted from celery to use admin configurable autoretry settings
+    """
```

Comment:
Please include a (github) link to the original source code.

Reply:
the function is removed